ZIGSIMPlus/Views/CommandSelectionTabNavigationController.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  CommandSelectionTabNavigationController.swift
+//  ZIGSIMPlus
+//
+//  Created by Takayosi Amagi on 2019/06/17.
+//  Copyright © 2019 Nozomu Kuwae. All rights reserved.
+//
+
+import UIKit
+
+class CommandSelectionTabNavigationController: UINavigationController {
+    var mediator: CommandAndServiceMediator?
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        if mediator == nil {
+            fatalError("CommandSelectionTabNavigatorController.mediator is not initialized")
+        }
+
+        // Initialize child view controllers
+        for viewController in viewControllers {
+            if type(of: viewController) == CommandSelectionViewController.self {
+                let vc = viewController as! CommandSelectionViewController
+                vc.presenter = CommandSelectionPresenter(view: vc, mediator: mediator!)
```

Comment:
Later I'd like to move these composing process into a certain class.

Reply:
sounds great!
I can't imagine how it should be because I don't understand the lifecycle of iOS classes yet... 😇